packages/components/src/List/Toolbar/SelectSortBy/SelectSortBy.component.js

```diff
@@ -75,15 +86,17 @@ function SelectSortBy({ field, id, isDescending, onChange, options, t }) {
 				</NavDropdown>
 			)}
 			{selected && (
-				<NavItem
-					id={id && `${id}-order`}
-					onClick={onChangeOrder}
-					aria-label={t('LIST_CHANGE_SORT_BY_ORDER', {
-						defaultValue: 'Change sort order. Current order: {{sortOrder}}.',
-						sortOrder: currentSortOrderLabel,
-					})}
-				>
-					{currentSortOrderLabel}
+				<NavItem>
+					<OrderChooser
```

Comment:
Using a sub component of another unrelated component feels a little wrong. It's a little hidden, so changes in the ResourcePicker will break the List.
Either externalize the order chooser, either just stay with a simple button here. (BTW the caret seems weird, indicating that it's a dropdown but on click it switch :/ )